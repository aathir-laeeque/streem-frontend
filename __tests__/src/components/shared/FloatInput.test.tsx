import { FloatInput } from '#components';
import React from 'react';
import { create, ReactTestRenderer } from 'react-test-renderer';

describe('<FloatInput />', () => {
  let snapshot: ReactTestRenderer;

  beforeAll(() => {
    const component = (
      <FloatInput
        placeHolder="FloatInput test"
        label="FloatInput Test"
        value=""
        id="FloatInputTestId"
        onChange={() => jest.fn()}
        required
        disabled
      />
    );
    snapshot = create(component);
  });

  it('should render and match the snapshot', () => {
    expect(snapshot.toJSON()).toMatchSnapshot();
  });
});
