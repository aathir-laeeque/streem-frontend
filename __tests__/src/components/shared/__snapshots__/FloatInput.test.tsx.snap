// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FloatInput /> should render and match the snapshot 1`] = `
<div
  className="sc-fzqNJr eMLkQT"
>
  <div
    className="wrapper"
  >
    <label
      className="on"
      style={
        Object {
          "color": "#666666",
          "opacity": 0.4,
        }
      }
    >
      FloatInput Test
    </label>
    <input
      className="input"
      data-testid="FloatInputTestId"
      disabled={true}
      name="FloatInputTestId"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      placeholder="FloatInput Test"
      style={
        Object {
          "borderColor": "#666666",
          "opacity": 0.4,
        }
      }
      type="text"
      value=""
    />
  </div>
</div>
`;
